lib/api/controllers/securityController.js

```diff
@@ -420,8 +420,6 @@ class SecurityController {
   deleteUser(request) {
     assertHasId(request);
 
-    const deleteGenerator = Bluebird.coroutine(securityDeleteUserGenerator);
-
     return this.kuzzle.repositories.user.delete(request.input.resource._id, {refresh: request.input.args.refresh})
       .then(() => this.kuzzle.repositories.token.deleteByUserId(request.input.resource._id))
       .then(() => deleteGenerator(this.kuzzle, request))
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Move the declaration of "deleteGenerator" before this usage. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)